modules/toggle.js

```diff
@@ -0,0 +1,11 @@
+const toggleCheckbox = (todoArray) => {
+  const chbox = document.querySelectorAll('input[type="checkbox"]');
+  chbox.forEach((check, i) => {
+    check.addEventListener('click', () => {
+      todoArray[i].completed = !todoArray[i].completed;
+      localStorage.setItem('todoArray', JSON.stringify(todoArray));
```

Comment:
I will suggest you make use of your `addData()` function to avoid repeating yourself so as to follow the best practices of [KISS](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md)

Reply:
noted on this but i faced an issue with importing the function in the toggle module and that's why I added items to the local storage directly. Thank for the feedback 👌